tests/files/environments/development/us-east-1.yml

```diff
@@ -0,0 +1,15 @@
+defaults:
+  parameters:
+    CidrBlock: '10.0'
+stacks:
+  - name: Test-Dev-VPC
+    template_name: VPC
+    parameters:
+      Name: "TestVPC"
+
+  - name: Test-Dev-SecurityGroup
+    template_name: SecurityGroup
+    parameters:
+      VpcId:
+        Stack: Test-VPC
+        Output: VpcId
```

Comment:
My vision for environments was a single file for each environment with a "default" region set in that environment file, then a separate file for each stack to allow for better automation of adding/removing stacks via supporting tools. Then you could override the region the stack lives in inside the stack file. Having a file per region seems a little limiting.

Structure I was envisioning:

```
environments/sandbox.yml # parameters, default region, no stacks — environment is inferred by existence of file
environments/sandbox/us-east-1.yml # just region parameters, no stacks
environments/sandbox/stacks/SecurityGroup.yml # can override region here, if different from environment
```